buidl/psbt.py

```diff
@@ -541,6 +546,194 @@ def serialize(self):
             result += psbt_out.serialize()
         return result
 
+    def describe_basic_multisig_tx(self, root_fingerprint_for_signing=None):
+        """
+        Describe a typical multisig transaction in a human-readable way for manual verification before signing.
+
+        This tool supports transactions with the following constraints:
+        * We sign ALL inputs with the same multisig wallet (quorum + root fingerprints).
+        * There can only be 1 output (sweep transaction) or 2 outputs (spend + change). If 2 outputs, we validate the change (it has the same quorum/xfp as the inputs we sign).
+
+        Advanced/atypical transactions will throw an error as these cannot be easily summarized.
+        An error does not mean there is a problem with the transaction, only that it cannot be trivially summarized.
+
+        If `root_fingerprint_for_signing` (a hex string) is included, then the root_paths needed to derive child private keys (for signing) from that seed will be returned.
+        """
+
+        self.validate()
+
+        TX_FEE_SATS = self.tx_obj.fee()
```

Comment:
why caps?

Reply:
This code block is so complicated, and my thinking is that by marking it as all caps I'll know that it was treated as a constant (not edited). Perhaps that's a bad convention?

Reply:
I usually reserve all-caps for global constants. What we have here is something that doesn't change within the function, which to me should be a normal variable. There may come some change in the future where the fee may need to be modified based on what condition from the taproot MAST tree gets executed, so I would change this to be a variable.